syndeo-template/src/main/java/com/google/cloud/syndeo/common/ProviderUtil.java

```diff
@@ -141,13 +142,24 @@ public static PCollectionRowTuple applyConfigs(
 
       if (tuple.getAll().containsKey(ERRORS_TAG)) {
         // TODO(pabloem): Deal with errors.
+        // TODO(dippatel): Remove hardcoded DLQ topic
         PCollectionRowTuple.of("input", tuple.get(ERRORS_TAG))
             .apply(
                 new SyndeoStatsSchemaTransformProvider()
                     .from(
                         SyndeoStatsSchemaTransformProvider.SyndeoStatsConfiguration.create(
                             "errors"))
                     .buildTransform());
+        if (spec.configuration.getString("dlqType") == "PUBSUB"
+            || spec.configuration.getString("dlq") != null) {
```

Comment:
Yes, "dlq" field will have the topic name. I wanted to keep it generic since we wanted to support more sinks in the future, "dlq_pubsub_topic" would be specific to pubsub. We would have to add more fields or change the name in the future. How about I change it to "dlqSink" instead?

Reply:
I think dlqSink might be a bit misleading and might convey dlqType. So depending on the sink, we might need different  options (like topic for pubsub, bucket for gcs, etc). I think having sink specific configs should be fine IMO.

Reply:
Because of the change from configuration parameters to top level parameters, this is irrelevant now. As now we will be including dlq information as any SchemaTransform at the top level. The parameter name will be directly taken from the relevant dlq SchemaTransform, pubsub in this case.